app/assets/javascripts/geoblacklight/downloaders/downloader.js

```diff
@@ -26,6 +26,9 @@
       this.downloading = true;
       var url = this.$el.data('downloadPath');
       this.options.spinner.show();
+      this.$el.removeAttr('href');
+      this.$el.prop('disabled', true);
+      this.$el.prop('innerHTML', 'Preparing download...');
```

Comment:
this:

1. removes the href attribute, which removes the link styling
2. disables the link (probably unnecessary, but it looks like existing JS code was re-enabling already when downloading was done)
3. changes the text of the link to indicate the download is processing

Reply:
So disabled doesn't work on href, this does work: https://github.com/geoblacklight/geoblacklight/pull/1404/commits/6e59a236526d7631e092eb7470fb999ca56b112e

Reply:
thanks - i'll make this change in my PR (and also remove the need to take away the href attr, which i don't think will be needed anymore

Reply:
we do want to take away the href, because that's the best way to indicate to screen reader users that it's not really a link. doing it with CSS won't tell them that, and adding `disabled` doesn't do anything.

Reply:
I would just remove all the disabled stuff from code. It is not necessary anymore when the spinner replaces the link.

Reply:
I didn't have the spinner replacing the link in this PR, it was going next to it.  But now I also realize that this PR is built on top of main, and that the JS changed.... so I think I need to go to the `modernize-js` branch and work off of that instead?

Reply:
It's kind of a tricky situation because that branch is a work in progress. But if you don't mind rebasing to that branch, it'd be super helpful